tests/test_SALSubsystems.py

```diff
@@ -10,7 +10,10 @@
 
 INDEX_ENUM_CHECK = re.compile(r"[^,= \w]+")
 
-VALID_LANGUAGES = {"CPP", "Java", "LabVIEW", "IDL", "Python"}
+# These are the "languages" we need to build libraries for. Note we removed
+# support for python runtime (previously SALPY) and the python/salobj
+# components will either use IDL (for DDS) or xml directly (for kafka).
+VALID_RUNTIME_LANGUAGES = {"CPP", "Java", "LabVIEW", "IDL"}
```

Comment:
Should LabVIEW still be mentioned here? It looks like you removed it everywhere from SALSubsystems.xml

Reply:
no, Labview is still quoted in a couple systems. Especially:

- MTMount (probably not needed any longer)
- ATMCS (still needed)
- ATPneumatics (still needed)
- MTM2 (probably not needed any longer)
- Test (no comments)

We may deprecated it when we go to kafka.